apps/dashr-wind-steaming/app.R

```diff
@@ -0,0 +1,443 @@
+
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(purrr)
+library(dplyr)
+library(DBI)
+library(RSQLite)
+library(glue)
+library(data.table)
+library(plotly)
+library(readr)
+library(VGAM)
+
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-opiod-endemic")
+setwd("/app/apps/dashr-wind-streaming")
```

Comment:
These `setwd` calls won't work on deployed apps, you can use something along the lines of:

```

appName <- Sys.getenv("DASH_APP_NAME")
if (appName != ""){
  pathPrefix <- sprintf("/%s/", appName)

  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)

  setwd(sprintf("/app/apps/%s", appName))
}
```

and

```

if (appName != "") {
  app$run_server(host = "0.0.0.0", port = Sys.getenv('PORT', 8050)) 
} else {
  app$run_server()
}
```